### ISSUE TITLE
Complete sandbox deployment of GeoPortal Server 2.0

### ISSUE THREAD

**Issue by torrin47** [Innovate-Inc/EDG_metadata#11]
Let's aim to complete this by 6/24/06.  If there are any challenges with ports or IIS, I'm happy to assist.

**Comment by torrin47** [comment]
Looks like they just released 2.0.1:
https://github.com/Esri/geoportal-server-catalog/releases/tag/v2.0.1
Let's go ahead and upgrade and just purge the existing elasticsearch index, start fresh.

**Comment by torrin47** [comment]
http://localhost:8080/geoportal/elastic/metadata/item/_search?from=0&size=10&sort=title.sort:asc

Here is the error

{"error":{"root_cause":[{"type":"search_parse_exception","reason":"No mapping found for [title.sort] in order to sort on"}],"type":"search_phase_execution_exception","reason":"all shards failed","phase":"query","grouped":true,"failed_shards":[{"shard":0,"index":"metadata","node":"5V6FEncgSwyRQVGnMJ3dHA","reason":{"type":"search_parse_exception","reason":"No mapping found for [title.sort] in order to sort on"}}]},"status":400}

Here is the response body

{"aggregations":{"id_uniqName_7_2":{"geohash_grid":{"field":"envelope_cen_pt","precision":3}},"id_uniqName_7_3":{"nested":{"path":"timeperiod_nst"},"aggregations":{"dates":{"date_histogram":{"field":"timeperiod_nst.begin_dt","interval":"year","min_doc_count":1}},"endDates":{"date_histogram":{"field":"timeperiod_nst.end_dt","interval":"year","min_doc_count":1}}}},"id_uniqName_7_4":{"date_histogram":{"field":"sys_modified_dt","interval":"day","min_doc_count":1}},"id_uniqName_7_5":{"terms":{"field":"sys_owner_s"}},"id_uniqName_7_6":{"terms":{"field":"apiso_TopicCategory_s"}},"id_uniqName_7_7":{"terms":{"field":"sys_metadatatype_s"}},"id_uniqName_7_8":{"terms":{"field":"contact_organizations_s"}},"id_uniqName_7_9":{"terms":{"field":"keywords_s","size":20}}}}

Torrin will escalate Errors to Esri

**Comment by torrin47** [comment]
I stopped ElasticSearch, purged the "Data" folder, and restarted ElasticSearch to purge the problematic ISO record.  When I tried testing GeoPortal, however, nothing was responding on port 8080 - I suspect that Kris rebooted the server and possibly updated the firewall rules, because port 8080 was being blocked by the firewall.  I installed Tomcat as a service and opened up the firewall, but I'm not sure leaving port 8080 wide open with a default installation of both Tomcat and GeoPortal is a great idea, so I ended up locking access to port 8080 down by IP - easy security.  If anyone wants to test, they can send their IP to us and we can give them access.
